tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt

```diff
@@ -38,6 +38,68 @@
 # This is an optional comment which can be used to give the name of the locale,
 # or any other organization-specific information such as tiers (T0, T1, …)
 
+Airbnb ; ar ; modern ; T2 Arabic
+Airbnb ; az ; modern ; T3 Azerbaijani
+Airbnb ; bg ; modern ; T3 Bulgarian
+Airbnb ; ca ; modern ; T2 Catalan
+Airbnb ; cs ; modern ; T2 Czech
+Airbnb ; da ; modern ; T2 Danish
+Airbnb ; de ; modern ; T1 German
+Airbnb ; el ; modern ; T2 Greek
+Airbnb ; en ; modern ; T0 American English
+Airbnb ; en_AU ; modern ; T1 English (Australia)
+Airbnb ; en_CA ; modern ; T1 English (Canada)
+Airbnb ; en_GB ; modern ; T1 English (Great Britain)
+Airbnb ; es ; modern ; T1 Spanish (Spain)
+Airbnb ; es_MX ; modern ; T1 Spanish (Mexico)
+Airbnb ; es_419 ; modern ; T2 Spanish (Latin America)
+Airbnb ; et ; modern ; T3 Estonian
+Airbnb ; fi ; modern ; T2 Finnish
+Airbnb ; fr ; modern ; T1 French
+Airbnb ; fr_CA ; modern ; T2 French (Canada)
+Airbnb ; ga ; modern ; T3 Gaelic
+Airbnb ; he ; modern ; T2 Hebrew
+Airbnb ; hi ; modern ; T3 Hindi
+Airbnb ; hr ; modern ; T3 Croatian
+Airbnb ; hu ; modern ; T2 Hungarian
+Airbnb ; hy ; modern ; T3 Armenian
+Airbnb ; id ; modern ; T3 Indonesian
+Airbnb ; is ; modern ; T3 Icelandic
+Airbnb ; it ; modern ; T1 Italian
+Airbnb ; ja ; modern ; T1 Japanese
+Airbnb ; ka ; modern ; T3 Georgian
+Airbnb ; kn ; modern ; T3 Kannada
+Airbnb ; ko ; modern ; T1 Korean
+Airbnb ; lt ; modern ; T3 Lithuanian
+Airbnb ; lv ; modern ; T3 Latvian
+Airbnb ; mk ; modern ; T3 Macedonian
+Airbnb ; mr ; modern ; T3 Marathi
+Airbnb ; ms ; modern ; T3 Malay
+Airbnb ; nl ; modern ; T1 Dutch
+Airbnb ; nl_BE ; modern ; T3 Flemish
+Airbnb ; no ; modern ; T3 Norwegian
+Airbnb ; pl ; modern ; T2 Polish
+Airbnb ; pt ; modern ; T1 Portuguese (Brazil)
+Airbnb ; pt_PT ; modern ; T2 Portuguese (Portugal)
+Airbnb ; ro ; modern ; T3 Romanian
+Airbnb ; ru ; modern ; T1 Russian
+Airbnb ; sk ; modern ; T3 Slovak
+Airbnb ; sl ; modern ; T3 Slovenian
+Airbnb ; sq ; modern ; T3 Albanian
+Airbnb ; sr ; modern ; T3 Serbian
+Airbnb ; sr_Cyrl_ME ; modern ; T3 Serbian (Montenegro)
+Airbnb ; sv ; modern ; T2 Swedish
+Airbnb ; sw ; modern ; T3 Swahili
+Airbnb ; th ; modern ; T3 Thai
+Airbnb ; fil ; modern ; T3 Filipino
+Airbnb ; tr ; modern ; T2 Turkish
```

Comment:
@srl295 - Can we reorder fil so it's alphabetical?

Reply:
ah, tl -> fil

yes – want a new PR for it? i'm not sure all of the others are alphabetical though. some might be by tiers

Reply:
> ah, tl -> fil
> 
> yes – want a new PR for it? i'm not sure all of the others are alphabetical though. some might be by tiers

yes - sgtm! In the past it was by tier. I've been meaning to reorganize the others, but it's been low priority. I've just been adding things alphabetically for anything new.